include/kos/wdt_timer.h

```diff
@@ -0,0 +1,106 @@
+/* KallistiOS ##version##
+
+   include/kos/wdt_timer.h
+   Copyright (C) 2024 Paul Cercueil
+*/
+
+/** \file    kos/wdt_timer.h
+    \brief   Watchdog timer support.
+    \ingroup kthreads
+
+    This file contains the watchdog timer API. A watchdog timer is a timer that
+    will trigger an action (through a pre-registered callback) after a timeout
+    expires. The code using a watchdog timer will typically reset it
+    periodically to prevent the action from happening.
+
+    \author Paul Cercueil
+
+    \see    kos/thread.h
```

Comment:
```suggestion
    \see    kos/thread.h
    \see    kos/worker_thread.h
```

Reply:
I felt like this should also be in the "see also" list, because it's technically the mechanism through which this is implemented and should go with the group. :)